client/Models/README.md

```diff
@@ -0,0 +1,37 @@
+# Models
```

Comment:
The model for a Panel/Group is a hook, but really should be a private one - ie `useTopics` (I would suggest a rename to `useTopicModel` to be hyper picky) would be defined in a `Model.ts` file in the Topics Panel (and not exported as it is private/internals). 

As for testing, Panels are meant to be behaviourally tested. Ie, a table with data appears with the expected info. As there is no UI (yet), thus no table, it makes asserting (indirectly) that the model is behaving abit more obtuse. I think what would be sensible is to add a placeholder `View.patternfly.ts`/`View.carbon.ts`, which just stringifies the model output. Then, tests can assert the right data is present (just not in a table), which can then be updated when the component work comes along

Reply:
Just to confirm - models live inside the component that needs them, whereas reusable hooks go in the `hooks` dir?

Therefore, `useTopics` is a particular hook (if we want to encapsulate for reuse) - and the `Model` can either call `useTopics` or just the default `useQuery`?

Reply:
> Just to confirm - models live inside the component that needs them, whereas reusable hooks go in the `hooks` dir?
> 
> Therefore, `useTopics` is a particular hook (if we want to encapsulate for reuse) - and the `Model` can either call `useTopics` or just the default `useQuery`?

Yes - absolutley. If the intent was to have a reusable hook representing/abstracting the query, than that should be in `Hooks`. Given the context (and other changes), I reviewed/interpreted it as a model hook

Reply:
Rearranged/refactored the code into a private hook in the Topics panel component.